zetaclient/zetacore/client.go

```diff
@@ -52,6 +52,9 @@ type Client struct {
 	// blocksFanout that receives new block events from Zetacore via websockets
 	blocksFanout *fanout.FanOut[ctypes.EventDataNewBlock]
 
+	// inboundBallotsOutOfGas maps the failed ballot identifier to the gas limit used to broadcast the vote
+	inboundBallotsOutOfGas map[string]uint64
```

Comment:
Can we rename `readyToExecuteInboundBallots`
out of gas mean that they're ready to be executed and need more gas

Reply:
It does mean they were executed before but need more gas, isn't it?